Web_Development/FrontEnd/CSS_web_fonts.md

```diff
@@ -0,0 +1,73 @@
+# CSS WEB FONTS
+
+CSS Web Fonts allow us to use fonts that are not installed locally. After choosing the the desired font, include the font file on the web server and you are good to go.
+This font can be defined within the CSS ` @font-face ` rule.
+
+```css
+@font-face {
+    /*Font Descriptors*/
+}
+```
+### Font Descriptors
+Font Descriptors are used to provide additional information about the font.
+
+Required Descriptors:
+- ` font-family `: Defines the name of the font
+- ` src `: Defines the URL to get the font 
+Optional Descriptors:
+- ` font-style `: 'italic', 'oblique' or 'normal'
+- ` font-weight `: Boldness of the font ('norma;', 'bold', 100-900)
+- ` font-stretch `: 'normal', 'condensed', 'expanded' ('condensed' and 'expanded' have ultra, extra and semi variants)
+
+Example: Web fontS in HTML
+
+```html
+<!DOCTYPE html>
+<html>
+	
+<head>
+	<style>
+		@font-face{
+				
+			font-family:monospace;   	/* Set font family to monospace */
+			src:url(sansation_light.woff);		}
+		
+		* {
+			font-family:monospace;
+			font-style:italic;       /** font style to italic */      }
+    
```

Comment:
Remove so many indentations, please write in a proper manner. The final output doesn't look good.

![image](https://user-images.githubusercontent.com/75671152/140874851-713b1087-1b2a-411f-b510-28523622dba1.png)